client/src/components/Post/Post.tsx

```diff
@@ -248,12 +299,46 @@ class Post extends Component<Props, State> {
                     <div className={'post-comments-container'}>
                         {
                             this.state.comments.map((comment, index) => (
-                                <div key={index} className={'comment-card'}>
+                                <div key={index} className={'comment-card'}
+                                     onMouseEnter={() => this.setIsShown(index)}
+                                     onMouseLeave={() => this.setIsShown(-1)}>
                                     <div className={'comment-header'}>
                                         {comment.username + ' - ' + comment.creationDate + ' '}
-                                        {comment.editDate && <span style={{color: '#818181'}}>edited</span>}
+                                        {comment.editDate && <span style={{color: '#818181', marginLeft: 5}}>edited</span>}
+                                        {
+                                            (parseJwt(localStorage.getItem('token')).role.includes('ROLE_ADMIN') ||
+                                                parseJwt(localStorage.getItem('token')).userId == this.state.post.userId)  && this.state.hoverIndex==index &&
+                                                <MdDelete
+                                                    color={'#DF6052'}
+                                                    size={20}
+                                                    onClick={() => this.setState({showDeleteCommentModal: true, commentToDelete: comment.content, commentToDeleteId:comment.id})}
+                                                />
+                                        }
+                                        {
+                                            (parseJwt(localStorage.getItem('token')).role.includes('ROLE_ADMIN') ||
+                                                parseJwt(localStorage.getItem('token')).userId == this.state.post.userId) && this.state.hoverIndex==index &&
```

Comment:
@Nestiii edit comment options should only appear for **comment owners**, not post owners nor admins. Per user stories:

`Como un​ usuario autor de un comentario, quisiera​ poder editar un comentario, de modo que​ quede guardado el comentario editado`

Align w/ Sofi (very similar correction above)